### ISSUE TITLE
no module named googleapiclient.discovery

### ISSUE THREAD

**Issue by Swifty-Sousa** [gsuitedevs/python-samples#46]
## Expected Behavior
takes to url to accept

## Actual Behavior
thrown this error "  File "quickstart.py", line 2, in <module>
    from googleapiclient.discovery import build
ImportError: No module named googleapiclient.discovery"

url given in documentation incase Quickstart fails also does not work

## Steps to Reproduce the Problem

1.run Quickstart.py
1.
1.

## Specifications

- Python version (`2.7.10`)
- OS (Mac)

**Comment by erickoledadevrel** [comment]
Which sample are you running? Have you installed the Python client library as advised in the guide?

https://developers.google.com/calendar/quickstart/python#step_2_install_the_google_client_library

**Comment by Swifty-Sousa** [comment]
yes after using the exact pip installation and putting the credentials.json in the same directory as Quickstart.py I get this error, moreover the link provided in Quickstart.py is dead and leads to an empty page.

**Comment by erickoledadevrel** [comment]
Which quickstart specifically are you trying to run? I linked to the Calendar API one, but I don't know which API you are trying to use.

**Comment by Swifty-Sousa** [comment]
I am trying to use the gmail api
